docs/concepts.rst

```diff
@@ -735,6 +737,55 @@ values by "case" instead of by entity.
 Other Features
 --------------
 
+Parallel Execution
```

Comment:
How do you feel about adding `.. versionadded:: 0.8.0` for parallel execution?

Reply:
Oh, now that we release notes, I wonder if we even need any of these `versionadded`. Users can check the release notes for when the feature or concept was released.

Reply:
This still sounds like a good idea to me, although I could go either way.